src/components/views/ViewCampaign.jsx

```diff
@@ -110,6 +116,23 @@ const ViewCampaign = ({ match }) => {
     );
   };
 
+  const loadDonations = campaignId => {
+    if (campaignId) {
+      CampaignService.getDonations(
+        campaignId,
+        0,
+        0,
+        (_donations, _donationsTotal) => {
+          setDonationsTotal(_donationsTotal);
+        },
+        err => {
+          ErrorHandler(err, 'Some error on fetching campaign donations, please try later');
+        },
+        Donation.COMMITTED,
```

Comment:
When user do a donation, a new donation with Pending status is created. That one should be included in donation count  for better interactivity. I mean user should see the new badge immediately after donating. So the default condition of `{ $ne: Donation.FAILED }` in getDonations is enough and that parameter is not necessary.